server/core/templates/base_htmx.html

```diff
@@ -20,6 +20,8 @@
 		<script src="/pkg/external/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz"></script>
 		<script src="/pkg/external/htmx.min.1.9.12.js" integrity="sha384-ujb1lZYygJmzgSwoxRggbCHcjc0rB2XoQrxeTUQyRjrOnlCoYta87iKBWq3EsdM2"></script>
 		<link rel="stylesheet" href="/pkg/style.css" />
+		<script src="/pkg/external/cred_update.js"></script>
+		<script src="/pkg/external/base64.js" async></script>
```

Comment:
These scripts should be imported in the places that require them, not everywhere